### PR TITLE
Some small improvements

### PR DESCRIPTION
**Pull request by UnknownPlatypus**
Hi !

Thx for the work on this project, this is a really straightforward yet super powerful library.
I was currently trying to use it for a project and had some ideas for improvements that this MR addresses: 

- 1.2x speed improvement by avoiding using zip/repeat in [send_message](https://github.com/xtrinch/fcm-django/blob/master/fcm_django/models.py#L162).
- Adding a `DeviceType` choice field for ease of use.
- Removing `.exists()` on [get_registration_ids](https://github.com/xtrinch/fcm-django/blob/e5b2e6528d81dc5cce6b07494b49dedbb77c0b58/fcm_django/models.py#L101) to prevent duplicate SQL request.
- fix a typo in doc and missing default for `UPDATE_ON_DUPLICATE_REG_ID`.

Let me know if this looks good to you.
I can update / remove any of the above following what you think is relevant.

Cheers

### CODE REVIEW

File: fcm_django/models.py [hunk 1]
```diff
@@ -266,9 +260,13 @@ def handle_topic_subscription(
 FCMDeviceManager = _FCMDeviceManager.from_queryset(FCMDeviceQuerySet)
 
 
-class AbstractFCMDevice(Device):
-    DEVICE_TYPES = (("ios", "ios"), ("android", "android"), ("web", "web"))
+class DeviceType(models.TextChoices):
```
Comment (UnknownPlatypus) [inline fcm_django/models.py:263]: Juste noticed you support django 2.2 so this might not be possible yet

Comment (xtrinch) [reply fcm_django/models.py:263]: No worries we can make this a major release and drop 2.2 django support

File: README.rst [hunk 2]
```diff
@@ -132,6 +132,7 @@ Edit your settings.py file:
         # Transform create of an existing Device (based on registration id) into
 		    # an update. See the section
         # "Update of device with duplicate registration ID" for more details.
+        # default: False
```
Comment (xtrinch) [inline README.rst:135]: Please amend in readme which django versions are still supported by this change

Comment (UnknownPlatypus) [reply README.rst:135]: btw, the `index.rst` file seems to not be exactly in sync with the `readme.rst`. Is there some kind of command to run to sync them or should I juste copy paste ?

Comment (xtrinch) [reply README.rst:135]: Just copy paste. It's not ideal I know :-)

**Review by xtrinch — APPROVED** [review]

**Comment by xtrinch** [comment]
@UnknownPlatypus thanks for the PR!

**Pull request merged**